src/main/resources/assets/moldraw/lang/en_us.json

```diff
@@ -0,0 +1,11 @@
+{
+  "tooltip.moldraw.shift_view": "§7Press Shift to view the structure",
+
+  "config.moldraw.option.enabled": "Enable Molecule Drawer",
```

Comment:
"Enable mod" feels more correct since it also enables the colors in formulae without molecules